packages/server/postgres/migrations/1644936872423_addTeamPromptResponseTable.ts

```diff
@@ -0,0 +1,72 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    DO $$
+    BEGIN
+      IF NOT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'Reactji') THEN
+        CREATE TYPE "Reactji" AS (shortName text, userId text);
+      END IF;
+
+      ALTER TYPE "DiscussionTopicTypeEnum" ADD VALUE 'teamPromptResponse';
+
+      ALTER TYPE "MeetingTypeEnum" ADD VALUE 'teamPrompt';
+
+      CREATE TABLE IF NOT EXISTS "TeamPromptResponse" (
+        "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+        "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+        "meetingId" VARCHAR(100) NOT NULL,
+        "userId" VARCHAR(100) NOT NULL,
+        "content" TEXT NOT NULL,
+        "plaintextContent" TEXT NOT NULL,
```

Comment:
+1 don't hesitate to tell me no, but if `content` is stored as JSON, then `plaintextContent` could be GENERATED and we won't need to do that at the application layer

Reply:
I haven't found any easy way to do it so I'll create a task for it and let's continue with doing that conversion in the application layer for now.

Reply:
> any chance you know a simple way of converting json to text in pg?

i think we talked about this in the 1:1. it'd have to be a function that traversed the json recursively picking out the text property on text nodes & aggregating them. definitely worth holding off for now

> I was thinking about it and I have one more question, is there any benefit of storing the same data in a different format in DB? what's the use case for that field, in general?

rich text is hard to query against, read for debugging, & no good for searching if we even wanted to implement something like elastic search